### DISCUSSION TITLE
What to do with working locations?

### DISCUSSION THREAD

**Discussion by SeSamSeg** [coedit/gcal-rainbow-surf#8] [Ideas]
Do we want to merge all day working locations?
I'm not sure what the best look would be.

**Comment by ezdub** [comment]
Hello! Thank you for starting the conversation. Will you please provide a screenshot of the area you would expect to be merged? 
Providing your steps taken to get to this point would be helpful as well.
Thank you.
-Z

**Reply by SeSamSeg** [reply]
We work with working locations in our team:
![image](https://github.com/coedit/gcal-rainbow-surf/assets/1176649/2fb26b96-9ac5-415b-a62a-342e1f1fa484)

But I now notice that they are already grouped in Home and Office 🤔And combining the icons doesn't add any room.
Maybe it's new or I didn't notice cause before using this plugin I never had everyone's calendar open at the same time.

**Reply by ezdub** [reply]
Ok, so if I follow you, it sounds like there's not a way we can improve the experience of merging matching events pertaining to working locations.
